pkgs/development/python-modules/quil/default.nix

```diff
@@ -53,13 +50,8 @@ buildPythonPackage rec {
     syrupy
   ];
 
-  disabledTests = [
-    # Syrupy snapshot needs to be regenerated
-    "test_filter_instructions"
-  ];
-
   meta = {
-    changelog = "https://github.com/rigetti/quil-rs/blob/${src.rev}/quil-py/CHANGELOG.md";
+    changelog = "https://github.com/rigetti/quil-rs/blob/quil-py/v${version}/quil-py/CHANGELOG.md";
```

Comment:
As this package's maintainer, I object.
Also, @r-ryantm will change it to `${src.tag}`.